presentation/common/src/test/content/Descriptor.test.ts

```diff
@@ -313,8 +329,22 @@ describe("Descriptor", () => {
         fields: [],
         filterExpression: "test filter",
       });
+      descriptor.fieldsFilterExpression = undefined; // eslint-disable-line deprecation/deprecation
```

Comment:
`fieldsFilterExpression` isn't deprecated, why do we need this eslint comment?

Reply:
Removed